crypto/asn1/a_gentm.c

```diff
@@ -108,7 +108,7 @@ ASN1_GENERALIZEDTIME *ASN1_GENERALIZEDTIME_adj(ASN1_GENERALIZEDTIME *s,
     return NULL;
 }
 
-const char *_asn1_mon[12] = {
+static const char *_asn1_mon[12] = {
```

Comment:
> static char _asn1_mon[12][4] = {

That would be actually best from segment placement viewpoint. And naturally `static const char`.